README.md

```diff
@@ -309,12 +321,36 @@ Add the version of `solc` you want to use into `index.html`:
 <script
   type="text/javascript"
   src="https://binaries.soliditylang.org/bin/{{ SOLC VERSION }}.js"
+  integrity="sha256-{{ BASE64-ENCODED HASH OF SOLC VERSION  }}"
```

Comment:
where is that `BASE64-ENCODED HASH OF SOLC VERSION` coming from?

Reply:
Yes. Maybe I can put in between underscore if that makes it easier to understand that it is just a placeholder.

Reply:
Honestly, for me the underscores make it look like an existing variable rather than a placeholder so I wanted to suggest removing them :P